packages/antd/src/FormItemAnt.tsx

```diff
@@ -31,7 +31,6 @@ export class FormItemAnt extends React.Component<BindFormItemAntProps> {
                 style={formStyle || undefined}
                 labelCol={labelCol}
                 wrapperCol={wrapperCol}
-                required={operation.required}
```

Comment:
`required` is a property of `Form.Item` and is needed to visually show the red star, which marks the field as required. [FormItem - Ant Design](https://ant.design/components/form/#Form.Item)